tests/utils/test_utils.py

```diff
@@ -226,39 +226,46 @@ def test_get_input_variables():
     pass
 
 
-@pytest.mark.skip(reason="needs updating")
 def test_get_s(spark):
     """
-    Tests that define_s() gives the gives the correct output when provided with
+    Tests that get_s() gives the gives the correct output when provided with
     appropriate inputs.
     """
-    pass
+    # Arrange
+    test_input_df = spark.createDataFrame(
+        [
+            (1, "GREGORY", "PARKIN", 1, None, "SAYED"),
+            (2, "ELIZABETH", "CARTER", 1, None, "SAYED"),
+            (3, "ALFONSO", None, 1, None, "SAYED"),
+            (1, "GREGORY", "PARKIN", 2, "AMICA", "MAGNUSSON"),
+            (2, "ELIZABETH", "CARTER", 2, "AMICA", "MAGNUSSON"),
+            (3, "ALFONSO", None, 2, "AMICA", "MAGNUSSON"),
+            (1, "GREGORY", "PARKIN", 3, "LIZ", "CARTER-JONES"),
+            (2, "ELIZABETH", "CARTER", 3, "LIZ", "CARTER-JONES"),
+            (3, "ALFONSO", None, 3, "LIZ", "CARTER-JONES"),
+        ],
+        ["id_df1", "fn_df1", "sn_df1", "id_df2", "fn_df2", "sn_df2"],
+    )
+    test_input_variables = {
+        "spark_session_size": "small",
+        "df1_path": "path_to_df_1",
+        "df2_path": "path_to_df_2",
+    }
 
+    expected_output = {
+        "spark_session_size": "small",
+        "df1_path": "path_to_df_1",
+        "df2_path": "path_to_df_2",
+        "s": 9,
+    }
```

Comment:
These could be simplified to just contain the key:value pairs that are relevant to the test, e.g.:

```suggestion
    test_input_variables = {}

    expected_output = {"s": 9}
```

Reply:
Agreed, extra detail doesn't really add anything and is not in keeping with the other tests.

Am going to test GitHub's "Commit suggestion" for this one.